### PR TITLE
18.0.0 release

### PR DESCRIPTION
**Pull request by agliga**
Merge to master for release

### CODE REVIEW

**Review by ArtBlue — APPROVED** [review]

**Comment by ianmcburnie** [comment]
Please separate out these 3 examples (to be consistent with rest of docs):

Screenshot 2024-06-27 at 9 04 30 AM

**Comment by ianmcburnie** [comment]
I thought we were removing this from docs (it can stay in storybook):

Screenshot 2024-06-27 at 9 16 37 AM

File: docs/_includes/chart-legend.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<div id="chart-legend">
+    {% include section-header.html name="chart-legend" version=page.module_metadata.chart-legend.ds-component.version %}
+
+    <p>A representation of legend keys and their values on the plotted area of any chart or graph.</p>
+    <div class="demo">
+        <div class="demo__inner">
+            <div style="width: 250px">
+                <div class="chart-legend">
+                    <dl aria-label="Example donut chart legend">
+                        <div class="chart-legend__item chart-legend__item">
```
Comment (ianmcburnie) [inline docs/_includes/chart-legend.html:10]: `chart-legend__item` class appears twice in every item.

File: docs/_includes/donut-chart.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    {% include section-header.html name="donut-chart" version=page.module_metadata.donut-chart.ds-component.version %}
+
+    <p>
+        The donut chart is a data visualization module designed to display comparative data in an easily digestible and
+        visually appealing format. Our donut chart is composed of four sections: the title, metric, graph, and
+        legend, each serving a distinct purpose to enhance the user's understanding of the data presented.
+    </p>
+    
+    <p>
+        In practice, the <span class="highlight">.donut-chart__graph</span> is where we will load in our Highcharts donut chart.
```
Comment (ianmcburnie) [inline docs/_includes/donut-chart.html:11]:
```suggestion
        In practice, the <span class="highlight">.donut-chart__graph</span> is where we will load in our 3rd party donut chart.
```

**Comment by ianmcburnie** [comment]
All code indentation should be like this:

Screenshot 2024-06-27 at 9 28 26 AM

Not this:

Screenshot 2024-06-27 at 9 28 31 AM

**Comment by ianmcburnie** [comment]
Should we show two examples of donut charts, one for each layout, with the small screen layout first? That would make the choice of DOM order understandable.

Screenshot 2024-06-27 at 9 43 28 AM

**Comment by agliga** [comment]
> I thought we were removing this from docs (it can stay in storybook):
> 
> Screenshot 2024-06-27 at 9 16 37 AM

I believe we had discussed to keep it. We had a meeting on this. Im fine to remove it though.

**Comment by agliga** [comment]
> All code indentation should be like this:
> 
> Screenshot 2024-06-27 at 9 28 26 AM
> Not this:
> 
> Screenshot 2024-06-27 at 9 28 31 AM

Fixed, however did not check every occurrence.

File: docs/_includes/donut-chart.html (hunk 2 above)
Comment (agliga) [reply docs/_includes/donut-chart.html]: Done

**Comment by montoya332** [comment]
> For the record, this looks nasty to me. Poor textbox, way too much going in there. I hope this is a rare UI scenario.
> 
> Screenshot 2024-06-27 at 9 07 14 AM

agree , is it just me or do the icons/text also seem misaligned

**Review by ArtBlue — APPROVED** [review]
Looks great!

**Pull request merged**